fenix/app/src/main/java/org/mozilla/fenix/HomeActivity.kt

```diff
@@ -251,12 +258,11 @@ open class HomeActivity : LocaleAwareAppCompatActivity(), NavHostActivity {
 
         maybeShowSplashScreen()
 
-        // There is disk read violations on some devices such as samsung and pixel for android 9/10
-        components.strictMode.resetAfter(StrictMode.allowThreadDiskReads()) {
```

Comment:
I _think_ that this was triggered by reading from `settings` on the main thread to fetch the cached PBM state and that it's safe to remove, but it's worth keeping an eye on.